tck/api/src/main/java/org/eclipse/microprofile/metrics/tck/cdi/MetricProducerFieldBean.java

```diff
@@ -13,11 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.astefanutti.metrics.cdi.se;
+package org.eclipse.microprofile.metrics.tck.cdi;
 
 import org.eclipse.microprofile.metrics.Counter;
 import org.eclipse.microprofile.metrics.Gauge;
 import org.eclipse.microprofile.metrics.annotation.Metric;
+import org.eclipse.microprofile.metrics.tck.util.FooQualifier;
```

Comment:
Is there another change somewhere that uses FooQualifier?

Reply:
nm.  I see it's because we're no longer in the same package as FooQualifier.

Reply:
This import had to be added because FooQualifier was moved to a different package (before this commit, FooQualifier is in the same package as this test; after this commit, it is in a different package than this test), otherwise nothing changed